src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  let discount = 0;
+  const priceForDay = 40;
+  const count = days * priceForDay;
+
+  if (days >= 3) {
+    discount = 20;
+  }
+
+  if (days >= 7) {
+    discount = 50;
```

Comment:
The number of days that qualifies for a discount should also be defined as constants. This will make the code self-documenting and easier to change in the future if needed.